pkgs/top-level/all-packages.nix

```diff
@@ -2877,7 +2877,9 @@ in
     inherit (darwin.apple_sdk.frameworks) CoreBluetooth ForceFeedback IOKit OpenGL;
   };
 
-  doomseeker = qt5.callPackage ../applications/misc/doomseeker { };
+  doomseeker = qt5.callPackage ../applications/misc/doomseeker {
+    inherit (stdenv.cc) isClang;
```

Comment:
it would be simpler to just use `stdenv.cc.isClang` in the expression?

Reply:
I'll leave it up to you.  Since this package is supposed to be built with a call to the `mkDerivation inside `qt5`, and is NOT supposed to use `stdenv.mkDerivation`, I think it's a little clearer to leave `stdenv` out of scope entirely and just pass in the boolean, but I don't feel too strongly one way or the other.